docs/docs/guides/repository.mdx

```diff
@@ -247,7 +247,7 @@ Unlike `Repository`, Infrahub does not automatically update `Read-only Repositor
   mutation {
     CoreReadOnlyRepositoryUpdate(
       data: {
-        id: "ID_OF_THE_REPOSITORY"
+        hfid: ["My Git repository"]
```

Comment:
@wvandeun are you sure we need to put the HFID inside brackets? earlier in this doc there is an example with credentials > repository using HFID that looks like this: `credential: {hfid: "my-git-credential"}`. I think I tested back then and it was working fine

Reply:
You don't have to use brackets as long as the HFID is made up of a single field. You must use brackets when the HFID is made up of multiple fields. 

My personal opinion is that we should keep things simple and that we should only have one way of doing something. So I prefer to always use brackets. 

Not sure if there's any other opinions on this. Maybe should even remove the variant where you don't have to provide the brackets. IMO it is only providing confusion for users.